config/api.yml

```diff
@@ -621,6 +621,12 @@
         :identifier: storage_tag
       - :name: unassign
         :identifier: storage_tag
+  :ems_events:
```

Comment:
@abellotti it might help to have a quick rundown on how to add new subcollections to the MIQ API.  Is there a doc or something that the Lenovo crew can look over?